.changeset/pretty-forks-fly.md

```diff
@@ -0,0 +1,24 @@
+---
+'@backstage/core-components': patch
+'@backstage/plugin-adr': patch
+'@backstage/plugin-api-docs': patch
+'@backstage/plugin-azure-devops': patch
+'@backstage/plugin-azure-sites': patch
+'@backstage/plugin-bazaar': patch
+'@backstage/plugin-cicd-statistics': patch
+'@backstage/plugin-cost-insights': patch
+'@backstage/plugin-firehydrant': patch
+'@backstage/plugin-fossa': patch
+'@backstage/plugin-gcp-projects': patch
+'@backstage/plugin-git-release-manager': patch
+'@backstage/plugin-home': patch
+'@backstage/plugin-lighthouse': patch
+'@backstage/plugin-search': patch
+'@backstage/plugin-search-react': patch
+'@backstage/plugin-sonarqube': patch
+'@backstage/plugin-tech-radar': patch
+'@backstage/plugin-techdocs': patch
+'@backstage/plugin-user-settings': patch
+---
+
+Enable the eslint-plugin-react for non-test files and forbid `<span>` tag in favor of `<Typography component='span'>`.
```

Comment:
The main change here was the linter rule to encourage the Typography component instead of raw `span`, so would this work?: ``Add linter rule to encourage usage of MUI `<Typography>` over raw `<span>` ``

Reply:
+1

Reply:
Done